addons/project/models/rating.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models
+
+
+class Rating(models.Model):
+
+    _inherit = "rating.rating"
+
+    parent_rating = fields.Boolean(compute=lambda f: None, search="_search_parent_rating", string="search on parent rating object")
+
+    def _search_parent_rating(self, operator, value):
+        return self.env[self.env.context['model']]._model_domain()
```

Comment:
Having a "model" key in the context : 
- Is difficult to grep
- Will probably be erased by another piece of code that add unclear context keys.